packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -30,6 +33,8 @@ const SignUpPasswordView = FormView.extend({
 
   events: assign({}, FormView.prototype.events, {
     'click .use-different': preventDefaultThen('useDifferentAccount'),
+    'keyup #vpassword': '_onConfirmPasswordKeyUp',
+    'blur #vpassword': '_onConfirmPasswordBlur',
```

Comment:
I added check on blur because if user tabs to age input and the delay hasn't check it, it will automatically show the tooltip error. It seemed to be a little smoother.